source/clear-linux/get-started/bootable-usb/bootable-usb-windows.rst

```diff
@@ -4,18 +4,18 @@ Create a bootable USB drive on Windows
 ######################################
 
 Follow these instructions to create a bootable |CLOSIA| USB drive.
-Use an **8GB** or larger USB drive.
+Use an **8GB** or larger USB drive.Download either a live image, 
+``clear-<version>-live.img.xz`` or an installer image, 
+``clear-<version>-installer.img.xz``, from our `image`_ download page.
 
-These instructions assume you have
-:ref:`downloaded an appropriate image<download-usb-image>`.
-
-Alternative instructions for other operating systems are available:
+Instructions are also available for other operating systems:
 
 * :ref:`bootable-usb-mac`
 * :ref:`bootable-usb-linux`
 
 .. include:: ../../guides/maintenance/image-types.rst
-   :start-after: image-types-content:
+   :start-after: for different platforms and environments. 
+   :end-before: Table 1 lists the currently available images.
 
 .. include:: ../../guides/maintenance/download-verify-uncompress-windows.rst
```

Comment:
This didn't show up in the rendered file

Reply:
@tpleavitt The ReST includes don't show up in the rendered file on GitHub. 

Please compare the linux version that I pushed in PR 145. While that 'include' did not show up in rendered file on GitHub, it does however, show up on the build.  
Compare: 
https://github.com/mvincerx/clear-linux-documentation/blob/e33e17c584ae706fad2419c6b4b32d8d36f7fee4/source/clear-linux/get-started/bootable-usb/bootable-usb-linux.rst

With: 
https://clearlinux.org/documentation/clear-linux/get-started/bootable-usb/bootable-usb-linux

Reply:
@tpleavitt Please see second commit below.